src/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -392,6 +396,11 @@ public void setPerTargetOutput(final File perTargetOutput) {
             this.perTargetOutput = perTargetOutput;
         }
 
+        /** If set, the metrics collector will output per target coverage information to this file. */
```

Comment:
I rephrased both to:
`        /** Sets the (optional) File to write per-[base/target] coverage information to. If null (the default), no file is produced. */`